tests/test_config.py

```diff
@@ -133,3 +133,27 @@ def test_get_value(monkeypatch, key, value, expected_value_type_in_config):
     casted_value = get_value(key, type_=expected_value_type_in_config)
 
     assert isinstance(casted_value, expected_value_type_in_config)
+
+
+@pytest.mark.parametrize(
+    "kwargs, expected",
+    [
+        ({"app_name": "test_app", "api_url": "http://example.com"}, "/event"),
+        ({"app_name": "test_app", "api_url": "http://example.com/api"}, "/api/event"),
+        ({"app_name": "test_app", "event_namespace": "/event"}, "/event"),
+        ({"app_name": "test_app", "event_namespace": "event"}, "/event"),
+    ],
+)
+def test_event_name_space(mocker, kwargs, expected):
```

Comment:
```suggestion
def test_event_namespace(mocker, kwargs, expected):
```

nit for grep-ability